### PR TITLE
Add menu for clearing temperature calibration

### PR DESCRIPTION
**Pull request by pianistrevor**
Implements #224

### CODE REVIEW

File: src/Devices/TempProbe_TC.cpp [hunk 1]
```diff
@@ -89,3 +89,17 @@ void TempProbe_TC::setCorrection(float value) {
...
+ * clearCorrection(f)
+ *
+ * Sets the correction offset back to 0
+ */
+void TempProbe_TC::clearCorrection() {
+  COUT("old = " << correction);
+  if (correction != 0) {
+    correction = 0.0;
+    EEPROM_TC::instance()->setCorrectedTemp(correction);
+    serial(F("Set temperature correction to %i.%02i"), (int)correction, (int)(correction * 100 + 0.5) % 100);
```
Comment (jgfoster) [inline src/Devices/TempProbe_TC.cpp:103]: You could just say that the new correction is "0.0", but this is fine as well.

**Review by jgfoster — APPROVED** [review]
LGTM. Let's ask @prestoncarman for a review as well...

File: src/Devices/TempProbe_TC.cpp (hunk 1 above)
Comment (pianistrevor) [reply src/Devices/TempProbe_TC.cpp:103]: I like this because we would catch it if it’s bugged.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**